### DISCUSSION TITLE
Multi-select pre-selected question

### DISCUSSION THREAD

**Discussion by maicWorkGithub** [quasarframework/quasar#12513] [General - Components / Directives / etc] [answered]
https://codepen.io/maicworkgithub/pen/yLPvYGx?editors=1010

How to change pre-selected value to show it's label?
Above demo, select `apple`, it shows `Facebook 苹果`, very weird right？

**Comment by MilosPaunovic** [comment, marked as answer]
The key here is to add `emit-value` prop to your `QSelect`, so it would emit only the value of the option, not the entire option object.

[https://codepen.io/milospaunovic/pen/BamYjxO?editors=1010](https://codepen.io/milospaunovic/pen/BamYjxO?editors=1010)

**Reply by maicWorkGithub** [reply]
Emmm, I want component loaded with selected value `脸书`, not `Facebook`.
I kow ` modelMultiple: ref([{label: '脸书', value: 'Facebook'}]),` will works fine.
But label is an UI value, not a model, I dont want write it to my model, like an value get from server, I need walk through all value, and give them a label

**Reply by MilosPaunovic** [reply]
Then you'll need to add `map-options` prop, as well.

[https://codepen.io/milospaunovic/pen/BamYjxO?editors=1010](https://codepen.io/milospaunovic/pen/BamYjxO?editors=1010)

**Reply by maicWorkGithub** [reply]
WOW, thanks a lot!!
